js_modules/dagit/packages/core/src/runs/RunActionButtons.tsx

```diff
@@ -43,30 +41,28 @@ const CancelRunButton: React.FC<{run: RunFragment | undefined; isFinalStatus: bo
         });
       }
     },
-    [run?.id],
+    [runId],
   );
 
-  if (!run) {
+  if (!runId) {
     return null;
   }
 
   return (
     <>
-      {!isFinalStatus ? (
-        <Button
-          icon={<Icon name="cancel" />}
-          intent="danger"
-          disabled={showDialog}
-          onClick={() => setShowDialog(true)}
-        >
-          Terminate
-        </Button>
-      ) : null}
+      <Button
+        icon={<Icon name="cancel" />}
+        intent="danger"
+        disabled={showDialog}
+        onClick={() => setShowDialog(true)}
+      >
+        Terminate
```

Comment:
I'm not 100% sure, but I thought we were saying "cancel" for runs in the queued state and then "terminate" once the run has started? As long as this is consistent with the options offered in the dropdown on the run list page I think it's good!

Reply:
I believe it's "Terminate" everywhere. (It is in the run action menu.)